### PR TITLE
Zh trans / docs/reference/setup tools/kubeadm/kubeadm config.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by xichengliudui** [comment]
deploy logs：
ERROR 2018/12/15 09:44:52 docs/reference/setup-tools/kubeadm/kubeadm-config.md:73: unclosed shortcode

File: content/cn/docs/reference/setup-tools/kubeadm/kubeadm-config.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+{{% capture whatsnext %}}
+
+<!--
+* [kubeadm upgrade](/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/) to upgrade a Kubernetes cluster to a newer version
+{{% /capture %}}
+-->
+
+*  [kubeadm upgrade](/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/)将Kubernetes集群升级到更新版本[kubeadm upgrade]
+{{% /capture %}
```
Comment (xichengliudui) [inline content/cn/docs/reference/setup-tools/kubeadm/kubeadm-config.md]:
```suggestion
{{% /capture %}}
```

**Comment by chenrui333** [comment]
这个 PR 有mixed content 问题

**Comment by chenrui333** [comment]
I created a test PR, https://github.com/kubernetes/website/pull/11776

**Comment by chenrui333** [comment]
@congfairy The root cause with mixed content issue is your base branch is not to date. 

I think need to do:
- refresh your base branch (release-1.12) 
- rebase your work 
- then forced update your branch (PR issue would go)

This would be beneficial in the long run as well.

setup my local
```
git remote add congfairy git@github.com:congfairy/website.git
git fetch congfairy
git checkout -b zh_trans-/-docs/reference/setup-tools/kubeadm/kubeadm-config.md congfairy/zh_trans-/-docs/reference/setup-tools/kubeadm/kubeadm-config.md
```
rebase your work
```
git checkout release-1.12
git pull upstream release-1.12
git checkout zh_trans-/-docs/reference/setup-tools/kubeadm/kubeadm-config.md
git rebase -i release-1.12
git push origin zh_trans-/-docs/reference/setup-tools/kubeadm/kubeadm-config.md
```

**Comment by chenrui333** [comment]
The work has been verified in #11776.

I will put it on hold until you fixed your base branch issue.

**Comment by congfairy** [comment]
> 为什么删除 volume ？

发现有冲突 所以删除掉了。

**Comment by tengqm** [comment]
@congfairy Volume 那个文件为什么和你的 PR 有冲突？

**Comment by chenrui333** [comment]
volume 应该不会有冲突，我有个折中方案，干脆重新 branch out 做一个新的PR，这样会比较简单点。

**Comment by tengqm** [comment]
@chenrui333 我也是这样理解的。很可能是 git 命令使用不熟练造成的问题。在这个 PR 中并没有触及 volume.md 文件，因此不应该会产生冲突。所以我建议尝试 rebase 操作。可能是 rebase 过程中出了差错。

**Comment by chenrui333** [comment]
@tengqm 对的，我也觉得是的。

@congfairy 你可以试试看rebase，跟我上面列的command history 差不多。
https://github.com/kubernetes/website/pull/11712#issuecomment-447594514

**Comment by congfairy** [comment]
#11787

**Comment by chenrui333** [comment]
Thanks @congfairy

**Comment by tengqm** [comment]
Merged in another PR (see link above).

**Pull request closed without merging**